Projects/postcard

```diff
@@ -0,0 +1,36 @@
+# Postcard
+
+## Name
+postcard 1.0.4
+
+## Summary
+Postcard is a `#![no_std]` focused serializer and deserializer for Serde.
+
+Postcard aims to be convenient for developers in constrained environments, while allowing for flexibility to customize behavior as needed.
+## Goals
+1. Design primarily for `#![no_std]` usage, in embedded or other constrained contexts
+1. Support a maximal set of `serde` features, so `postcard` can be used as a drop in replacement
+1. Avoid special differences in code between communication code written for a microcontroller or a desktop/server PC
+1. Be resource efficient - memory usage, code size, developer time, and CPU time; in that order
+1. Allow library users to customize the serialization and deserialization behavior to fit their bespoke needs
+
+## Current State
+### Format Stability
+As of v1.0.0, `postcard` has a documented and stable wire format. More information about this wire format can be found in the `spec/` folder of the Postcard repository, or viewed online at https://postcard.jamesmunns.com.
+
+Work towards the Postcard Specification and portions of the Postcard 1.0 Release were sponsored by Mozilla Corporation.
+
+## Open tasks
+
+
+## Areas of research
+
+
+## Cross links to related topics/projects
+[Github link](https://github.com/jamesmunns/postcard)
+
+[crates.io link](https://crates.io/crates/postcard)
+
+[Postcard on Jamesmunns.com](https://postcard.jamesmunns.com/)
```

Comment:
```suggestion
[Postcard Format Specification](https://postcard.jamesmunns.com/)
```

Reply:
I made these edits in VSCode (again, I suppose), what happens if I push commit suggestion here?

Reply:
In general, git won't let you - there's a concept for "this changed in two places at the same time", usually called a "merge conflict". For now I'll just hit the "resolve conflict" button to dismiss this!